examples/torch/pearl_half_cheetah_vel.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+"""PEARL HalfCheetahVel example."""
+
+from garage.envs import GarageEnv, normalize
+from garage.envs.half_cheetah_vel_env import HalfCheetahVelEnv
+from garage.experiment import LocalRunner, wrap_experiment
+from garage.experiment.deterministic import set_seed
+from garage.experiment.meta_evaluator import MetaEvaluator
```

Comment:
nit: which of these can use a canonical import, e.g. `from garage.experiment import MetaEvaluator`?

Reply:
Only `LocalRunner` and `wrap_experiment`.

Reply:
I think MetaEvaluator is also available

@krzentner is there a reason you didn't export these TaskSamplers?

Reply:
Not in particular. It seemed likely we would move them at a later date.

Reply:
@lywong92 can you import them in `g/experiment/__init__.py` and update the imports for them to be canonical? 

examples are where most people will be getting their usage examples.